src/lando/api/legacy/app.py

```diff
@@ -46,7 +44,6 @@ def load_config() -> dict[str, Any]:
     """Return configuration pulled from the environment."""
     config = {
         "ALEMBIC": {"script_location": "/migrations/"},
-        "DISABLE_CELERY": bool(os.getenv("DISABLE_CELERY")),
```

Comment:
Do we not need to disable celery any more? I don't see the corresponding variable in the settings module.

Reply:
We've never used this variable as far as I can tell, and it doesn't provide any functionality that we need. When things were implemented manually in the "subsystem" (i.e., did not work out of the box) I think this made more sense to have. Now that we don't have to do anything extra to use Celery in Django, I don't think it makes sense to implement this unless the need for it arises.